.gitignore

```diff
@@ -52,3 +54,78 @@ tags
 # OSX
 src/impcnvgen.dSYM/
 src/optabgen.dSYM/
+
+# Generated D source
+src/access.d
+src/aggregate.d
+src/aliasthis.d
+src/apply.d
+src/argtypes.d
+src/arrayop.d
+src/arraytypes.d
+src/attrib.d
+src/builtin.d
+src/canthrow.d
+src/dcast.d
+src/dclass.d
+src/clone.d
+src/cond.d
+src/constfold.d
+src/cppmangle.d
+src/ctfeexpr.d
+src/declaration.d
+src/delegatize.d
+src/doc.d
+src/dsymbol.d
+src/denum.d
+src/errors.d
+src/expression.d
+src/func.d
+src/hdrgen.d
+src/id.d
+src/identifier.d
+src/imphint.d
+src/dimport.d
+src/dinifile.d
+src/inline.d
+src/init.d
+src/dinterpret.d
+src/json.d
+src/lexer.d
+src/link.d
+src/dmacro.d
+src/dmangle.d
+src/mars.d
+src/dmodule.d
+src/mtype.d
+src/nspace.d
+src/opover.d
+src/optimize.d
+src/parse.d
+src/sapply.d
+src/dscope.d
+src/sideeffect.d
+src/statement.d
+src/staticassert.d
+src/dstruct.d
+src/target.d
+src/dtemplate.d
+src/tokens.d
+src/traits.d
+src/dunittest.d
+src/utf.d
+src/dversion.d
+src/visitor.d
+src/lib.d
+src/nogc.d
+src/nspace.d
+src/color.d
+src/globals.d
+src/root/aav.d
+src/root/file.d
+src/root/filename.d
+src/root/speller.d
+src/root/man.d
+src/root/outbuffer.d
+src/root/response.d
+src/root/stringtable.d
```

Comment:
Currently the D source layout is as close to the C++ layout as possible.  If there's consensus I can change it.  Using `*.d` was discussed earlier, and has the slight disadvantage that new d files will not show up automatically.  Maintaining the gitignore list has not been difficult over the last year.